src/athena.js

```diff
@@ -139,7 +143,8 @@ export let executeQuery = async function({
     }
   },
   pollingDelay = 1000,
-  initPollingDelay = pollingDelay
+  initPollingDelay = pollingDelay,
+  maxRows = 10000
```

Comment:
shouldn't we start with a really low max (like 10), so as to force users to understand what they're doing?

Reply:
Giving a very low number would be a breaking change. I guess a more safe option would be to use Athena's default (1000 and they decided that it's fine). As I said, I want this function to be very simple for the user, so I would give a safely big number and only if you need more, you can tweak it.